src/services/Api.js

```diff
@@ -9,13 +9,11 @@ const client = axios.create({
 
 client.interceptors.request.use((request) => {
     const token = localStorage.getItem('jwt_token')
-    console.log(token)
-    console.log(request)
     if (token) {
         request.headers['Authorization'] = token;
```

Comment:
When using a JWT token the RFC specifies that the token should be prefixed with `Bearer `.